lib/govukFrontendPaths.js

```diff
@@ -0,0 +1,53 @@
+// core dependencies
+const path = require('path')
+
+// npm dependencies
+const fse = require('fs-extra')
+
+/**
+ * GOV.UK Frontend paths (cached)
+ *
+ * @type {Map<string[], govukFrontendPaths>}
+ */
+const govukFrontendPathsCache = new Map()
+
+/**
+ * Find GOV.UK Frontend via search paths
+ *
+ * @param {string[]} searchPaths - Search paths for `require.resolve()`
+ * @returns {{ baseDir: string, includePath: string, assetPath: string, config: { [key: string]: unknown } }}
+ */
+function govukFrontendPaths (searchPaths = []) {
+  if (govukFrontendPathsCache.has(searchPaths)) {
+    return govukFrontendPathsCache.get(searchPaths)
+  }
+
+  /**
+   * GOV.UK Frontend paths normalised
+   *
+   * v4.x - `/path/to/node_modules/govuk-frontend/govuk/all.js`
+   * v5.x - `/path/to/node_modules/govuk-frontend/dist/govuk/all.bundle.js`
+   */
+  const entryPath = require.resolve('govuk-frontend', { paths: searchPaths })
+  const dependencyPath = path.join('node_modules/govuk-frontend')
+  const baseDir = path.join(entryPath.split(dependencyPath)[0], dependencyPath)
+  const includeDir = path.dirname(entryPath)
+
+  govukFrontendPathsCache.set(searchPaths, {
+    baseDir,
+
+    includePath: `/${path.relative(baseDir, includeDir)}`,
+    assetPath: `/${path.relative(baseDir, path.join(includeDir, 'assets'))}`,
+
+    // GOV.UK Frontend plugin config
+    config: fse.readJsonSync(path.join(baseDir, 'govuk-prototype-kit.config.json'), {
+      throws: false
+    })
```

Comment:
@BenSurgisonGDS has suggested a new [`getScriptsAndAssetsConfig()` utility](https://github.com/alphagov/govuk-prototype-kit/pull/2319/files#diff-9aa82f0ed674e050695a7422b1cd56d43ce47e6953688a16a003bf49c3481622R269) for this

Couldn't find a "sync" way using an existing helper so might be a better option